taskvine/src/manager/vine_manager.c

```diff
@@ -4982,14 +4982,15 @@ static int poll_active_workers(struct vine_manager *q, int stoptime)
 
 	BEGIN_ACCUM_TIME(q, time_status_msgs);
 
-	int i, j = 1;
 	int workers_failed = 0;
 	// Then consider all existing active workers
-	for (i = j; i < n; i++) {
+	for (int i = 1; i < n; i++) {
```

Comment:
Let's leave the `int` outside the for, as that's the way we do it in the rest of the code.